test/format/format_config_def.c

```diff
@@ -323,9 +321,8 @@ CONFIG configuration_list[] = {
   {"stress.evict_reposition", "stress evict reposition",
     C_BOOL, 2, 0, 0, V_GLOBAL_STRESS_EVICT_REPOSITION},
 
-  {"stress.failpoint_eviction_fail_after_reconciliation", "stress failpoint eviction fail after "
-    "reconciliation", C_BOOL, 30, 0, 0,
-    V_GLOBAL_STRESS_FAILPOINT_EVICTION_FAIL_AFTER_RECONCILIATION},
+  {"stress.failpoint_eviction_fail_after_reconciliation", "stress failpoint eviction fail after reconciliation",
+    C_BOOL, 30, 0, 0, V_GLOBAL_STRESS_FAILPOINT_EVICTION_FAIL_AFTER_RECONCILIATION},
```

Comment:
Any idea what caused all of these whitespace reformats above? I don't see any changes or things that shortened them?

Reply:
Yes, different formatting styles.  I've opened WT-10793 to explain and "fix" this.

Reply:
Should be fixed.